Makefile

```diff
@@ -1,10 +1,28 @@
-SUBDIRS=kafka-base zookeeper kafka-statefulsets kafka-connect kafka-connect/s2i
+VERSION ?= latest
+
+SUBDIRS=kafka-base zookeeper kafka-statefulsets kafka-inmemory kafka-connect kafka-connect/s2i
 DOCKER_TARGETS=docker_build docker_push docker_tag
 
 all: $(SUBDIRS)
 $(DOCKER_TARGETS): $(SUBDIRS)
+release: release_prepare $(SUBDIRS) release_version release_pkg
+
+release_prepare:
+	rm -rf ./barnabas-$(VERSION)
+	rm -f ./barnabas-$(VERSION).tar.gz
+	mkdir ./barnabas-$(VERSION)
+	cp README.md ./barnabas-$(VERSION)
+
+release_version:
+	echo "Changing Docker image tags from :latest to :$(VERSION)"
+	find ./barnabas-$(VERSION)/ -name '*.yaml' -type f -exec sed -i '/image: "*enmasseproject\/[a-zA-Z0-9_-]*:latest"*/s/:latest/:0.1.0-rc1/g' {} \;
+	find ./barnabas-$(VERSION)/ -name '*.yaml' -type f -exec sed -i '/name: [a-zA-Z0-9_-]*IMAGE_TAG/{n;s/latest/0.1.0-rc1/}' {} \;
```

Comment:
Did you mean to hard code the `0.1.0-rc1` and not use $VERSION ?

Reply:
Of course I did mean to use VERSION :-( ... thanks for noticing.